clang/lib/Analysis/UnsafeBufferUsage.cpp

```diff
@@ -787,9 +915,8 @@ AST_MATCHER(FunctionDecl, isNormalPrintfFunc) {
 // Then if the format string is a string literal, this matcher matches when at
 // least one string argument is unsafe. If the format is not a string literal,
 // this matcher matches when at least one pointer type argument is unsafe.
-AST_MATCHER_P(CallExpr, hasUnsafePrintfStringArg,
-              clang::ast_matchers::internal::Matcher<Expr>,
-              UnsafeStringArgMatcher) {
+static bool hasUnsafePrintfStringArg(const CallExpr &Node, ASTContext &Ctx,
+                                     MatchResult &Result, llvm::StringRef Op) {
```

Comment:
nit: Within this file,  the string attached to a matched node is called a `Tag`.  So could you change `Op` to `Tag`?